pymc3/sampling.py

```diff
@@ -529,7 +534,7 @@ def _sample_population(draws, chain, chains, start, random_seed, step, tune,
 def _sample(chain, progressbar, random_seed, start, draws=None, step=None,
             trace=None, tune=None, model=None, **kwargs):
     skip_first = kwargs.get('skip_first', 0)
-    refresh_every = kwargs.get('refresh_every', 100)
+    # refresh_every = kwargs.get('refresh_every', 100)
```

Comment:
Why is this commented out?

Reply:
It's commented out because the variable is set but never referenced, and pylint complained. Arguably it should not be in this commit, but it seems too trivial to deserve one of its own.  I could put it back if you like.

I wondered if this was actually trying to set a global that I didn't know about, but I didn't find any explanation -- maybe it's obsolete code that should go, or I suppose I might have missed something important.  If I'm wrong, and it *is* important, we should probably try to make a test that establishes its importance.

Reply:
Ah, ok! I think this might have been left dangling after we removed live plots. Could you delete it?

Reply:
Done and pushed.